tests/functional_tests/0003_create_test.py

```diff
@@ -243,3 +257,22 @@ def test_14_create_rc_jail(release, jail, invoke_cli):
     )
 
     assert jail.running is False, 'Failed to stop rc_jail'
+
+
+@require_root
+@require_zpool
+@require_jail_ip
+def test_15_create_jail_with_assigned_ip(release, jail, invoke_cli, jail_ip):
+    jail = jail('assigned_ip_jail')
+
+    invoke_cli([
+        'create', '-r', release, '-n', jail.name, f'ip4_addr=vtnet0|{jail_ip}'
```

Comment:
hardcoded vtnet0 :)

Reply:
Yeah 😓 As we want to run tests only, we can keep the user interaction as less as possible as the only intention is to see if the functionality is working as desired. However, maybe we can take input of interface and if not supplied default to `vtnet0` ? Thoughts ?

Reply:
I think it's best to just drop vtnet0 entirely as iocage will handle that networking side of things with jail(8)